### ISSUE TITLE
Language modal should not jump when searching

### ISSUE THREAD

**Issue by jelly** [cockpit-project/cockpit#18681]
### Explain what happens

We should always use a fixed height (but not hardcoded).

![image](https://user-images.githubusercontent.com/67428/233420493-5f1edd75-6b0d-450b-a181-86703ca58d33.png)

@garrett This is what you had in mind right?

### Where is the problem in Cockpit?

None

### Server operating system

None

**Comment by garrett** [comment]
Yeah, it should be something that works in both desktop and mobile, probably like `min-height: max(50rem, 80vh);` (but this is just off the top of my head and might be wrong). And when things aren't matched, we need to say so with an empty state.

Probably something similar to PF Table's empty state, but super simplified — perhaps just the text (no icon, no link)... but it should be centered the the font treatment should be different from the rest of the list.

https://www.patternfly.org/v4/components/table/html-demos/#empty-state
https://www.patternfly.org/v4/components/empty-state/

**Comment by shubhit7** [comment]
Isn't this issue same as #18680 ?

**Comment by shubhit7** [comment]
@Garrett Thanks for your suggestions.

So there are two related issues, this one is about jumping.
Another is #18680 , which I am already working on.

In my opinion, all the above suggestions pertains to the issue #18680 and I can address them there.

Here, I can fix the jumping issue. Let me know if that sounds fine.
I was also wondering if it possible to combine these two issues, and create single PR which can fix both of these issues.

**Comment by garrett** [comment]
Ah, right; I was thinking of both issues together. (However, you do have the text in your video too.) Thanks!

**Comment by shubhit7** [comment]
Yeah, that's because I was fixing both the issues parallelly :)
Once the text issue is resolved and PR is merged, I will raise a PR here to fix the jumping issue.
